framework/Steps/RomTrainer.py

```diff
@@ -73,7 +74,9 @@ def _localInitializeStep(self,inDictionary):
       @ In, inDictionary, dict, the initialization dictionary
       @ Out, None
     """
-    pass
+    for output in inDictionary['Output']:
+      if isinstance(output, ROM):
```

Comment:
I think we only do Model.Validate for SingleRun and MultiRun, not for RomTrainer. I agree with you, we may do not need the if check, since the only output for RomTrainer is ROM, but it seems to me we do not have the mechanism to check it.